modules/ROOT/pages/release-notes.adoc

```diff
@@ -126,6 +126,21 @@ A page that is generated with this version is still compatible with a previous v
 For the users of Bonita Community edition, Bonita 2022.2 also comes with all the bug fixes released in the Maintenance versions of Bonita 2022.1 (up until version 2022.1-u6).
 The detailed list is available in the "What’s new in Bonita 2022.1" page, in the "Bug fixes" section.
 
+
+=== Fixes in Bonita 2022.2-u9 (2024-04-18)
+
+==== Fixes in Bonita Runtime including Bonita Applications
+
+* RUNTIME-48 - SFlowNodeNotFoundException: Quartz's Trigger associated to a Boundary timer does not get deleted when the process instance terminates
+* RUNTIME-1824 - Filter on "caller" does not work when searching for ArchivedProcessInstance
+* RUNTIME-1828 - ArchivedProcessInstance are not deleted when not root process instances
+* RUNTIME-1832 - Initialize Keycloak CryptoIntegration to fix `java.lang.RuntimeException: java.lang.IllegalStateException: Illegal state. Please init first before obtaining provider`
+* RUNTIME-1833 - SanitizerFilter should be disabled by default to avoid data loss when updating. Please visit the xref:security:sanitizer-security.adoc[HTML sanitizer filter] articale to evaluate the risks.
```

Comment:
```suggestion
* RUNTIME-1833 - SanitizerFilter should be disabled by default to avoid data loss when updating. Please visit the xref:security:sanitizer-security.adoc[HTML sanitizer filter] article to evaluate the risks.
```